CHANGES.rst

```diff
@@ -1179,6 +1181,9 @@ New Features
 
 - ``astropy.io.fits``
 
+  - Fix convenience functions (`getdata`, `getheader`, `append`, `update`) to
```

Comment:
This should also be under "bug fix", not "new features", if needed at all. I am not sure why @embray asked for duplicate entries. Usually, it is just under one section...